eagle-core/eagle-alert-parent/eagle-alert/alert-common/src/main/java/org/apache/eagle/alert/engine/model/AlertStreamEvent.java

```diff
@@ -39,6 +39,7 @@
     private long createdTime;
     private String category;
     private AlertSeverity severity = AlertSeverity.WARNING;
+    private boolean duplicationChecked = false;
```

Comment:
Just using `private boolean duplicationChecked;` is fine, because the default value of `boolean` in jdk is false.